drivers/src/mailbox.rs

```diff
@@ -70,10 +69,44 @@ impl Mailbox {
     /// * 'MailboxRecvTxn' - Object representing a receive operation
     pub fn try_start_recv_txn(&self) -> Option<MailboxRecvTxn> {
         let mbox = mbox::RegisterBlock::mbox_csr();
-        match mbox.status().read().mbox_fsm_ps() {
-            MboxFsmE::MboxExecuteUc => Some(MailboxRecvTxn::default()),
-            _ => None,
+        if mbox.status().read().mbox_fsm_ps().mbox_execute_uc() {
+            match mbox.status().read().status() {
```

Comment:
What is the purpose of this change?

Reply:
In the loop in `runtime/src/main.rs`, I'm continually calling `try_start_recv_txn` under the assumption that if there's a command in the mailbox and we're in execute state, we've received a new command.

The problem is that when we write data back to the mailbox, it's still in execute state. So until the SoC clears the execute register, runtime fw will keep looping and trying to execute the same command repeatedly.

I'm open to a different way of doing this though.

Reply:
In the RTL, once you write to mbox.status(), mbox.mbox_fsm_ps() switches from ExecuteUc to ExecuteSoc. Apparently this is not happening in the sw-emulator.

Reply:
I removed this. I still think the emulator needs to be updated, but that's not blocking this PR (mostly because the test steps until the status is set and then stops).

I'm happy to make this change in this PR or just do it in a follow-up.

Reply:
Resolving for now